app/src/main/java/com/android/sample/model/activity/ActivitiesRepositoryFirestore.kt

```diff
@@ -38,6 +38,8 @@ class ActivitiesRepositoryFirestore(private val db: FirebaseFirestore) : Activit
                       data["title"] as String,
                       data["description"] as String,
                       data["date"] as Timestamp,
+                      data["startTime"] as? String ?: "00:00",
```

Comment:
good implementation but could improve data["startTime"] by giving it a Timestamp.